From b83d154a61441031a5f185d4197e6124ad97f509 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 17 Jul 2018 08:26:53 +0800
Subject: [PATCH 4267/5242] MLK-18605-14 drm/imx: lcdif: adjust 'max_height'
 for '1080x1920' mode

commit  a3a00170a18aca38781806e4564bc2cebe011c64 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the maximum height limitation to 1920 to support
'1080x1920' resolution mode. It is a temporary work
around and will be improved later.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 44d0209e97e0c574af30dd7a7d7e059d4ddf996d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/lcdif/lcdif-crtc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/lcdif/lcdif-crtc.c b/drivers/gpu/drm/imx/lcdif/lcdif-crtc.c
index 9895544..91f09c7 100644
--- a/drivers/gpu/drm/imx/lcdif/lcdif-crtc.c
+++ b/drivers/gpu/drm/imx/lcdif/lcdif-crtc.c
@@ -297,7 +297,7 @@ static int lcdif_crtc_bind(struct device *dev, struct device *master,
 
 	/* limit the max width and height */
 	drm->mode_config.max_width  = 1920;
-	drm->mode_config.max_height = 1080;
+	drm->mode_config.max_height = 1920;
 
 	dev_set_drvdata(dev, lcdif_crtc);
 
-- 
1.7.9.5

